test_calculus.py

```diff
@@ -66,3 +66,35 @@ def test_trapezoid_scipy():
     Unit test for scipy implementation of trapezoid method
     '''
     assert np.isclose(calc.trapezoid_scipy(np.sin, 0, np.pi), 2)
+
+def d3(x):
```

Comment:
I don't see the usefulness of these functions

Reply:
I see that calc.adapt needs these to be defined, but they're alread defined in calculus.py